libs/langchain/langchain/tools/audio_utils.py

```diff
@@ -0,0 +1,18 @@
+import tempfile
+from pathlib import Path
+
+
+def save_audio(audio: bytes) -> str:
+    """Save audio to a temporary file and return the path."""
+    with tempfile.NamedTemporaryFile(mode="bx", suffix=".wav", delete=False) as f:
+        f.write(audio)
+    return f.name
+
+
+def load_audio(audio_file_path: str) -> bytes:
+    """Load audio from a file into bytes."""
+    if Path(audio_file_path).exists():
```

Comment:
`exists()` will return True if `audio_file_path` would be directory. Maybe it's better to use `.is_file()` ?